packages/nextjs/pages/index.tsx

```diff
@@ -45,6 +45,16 @@ const Home: NextPage = () => {
                 to build your own UI.
               </p>
             </div>
+            <div className="flex flex-col bg-base-100 px-10 py-10 text-center items-center max-w-xs rounded-3xl">
+              <MagnifyingGlassIcon className="h-8 w-8 fill-secondary" />
+              <p>
+                Explore your local hardhat node with the{" "}
```

Comment:
```suggestion
                Explore your local transactions with the{" "}
```

I know that "local hardhat node" is actually more accurate, but I feel this makes more sense for beginners. Anyway, it's just a soft suggestion :D